lib/factory.rb

```diff
@@ -0,0 +1,18 @@
+class Factory
+
+    def initialize
```

Comment:
Don't see your factory making facilities based on the dmv apis! This was part of the project requirements in iteration 3